tests/test_recipients.py

```diff
@@ -0,0 +1,76 @@
+import json
+from unittest.mock import patch
+from utils import CopyingMock
+from mock_requests_request import side_effect_recipients
+
+from tap_surveymonkey.surveys import SurveyId
+from tap_surveymonkey.collectors import CollectorId
+
+from tap_surveymonkey import main
+
+
+@patch('requests.request', new_callable=CopyingMock)
+def test_recipient(
```

Comment:
To be consistent with `test_sync_collectors`?
```suggestion
def test_sync_recipient(
```